.storybook/webpack.config.js

```diff
@@ -0,0 +1,11 @@
+module.exports = (storybookBaseConfig, configType, defaultConfig) => {
+    defaultConfig.module.rules.push({
+                test: /\.scss$/,
```

Comment:
Code formatting is wrong, you should use 2 spaces, right there you have mixed 4 with... a lot :D